lib/datasource/conan/index.ts

```diff
@@ -0,0 +1,117 @@
+import { logger } from '../../logger';
+import { Http } from '../../util/http';
+import { ensureTrailingSlash } from '../../util/url';
+import * as loose from '../../versioning/loose';
+import type {
+  DigestConfig,
+  GetReleasesConfig,
+  Release,
+  ReleaseResult,
+} from '../types';
+
+export const id = 'conan';
+export const customRegistrySupport = true;
+export const defaultRegistryUrls = ['https://conan.bintray.com/'];
```

Comment:
This appears to be deprecated? I'm not familiar with conan at all, but I found this documentation: https://docs.conan.io/en/latest/uploading_packages/remotes.html#conan-center-deprecated

Reply:
Ha! so it is.  Would you believe I started this pre deprecation? I'll get that fixed!